include/SH3/system/glprogram.hpp

```diff
@@ -48,9 +48,11 @@ namespace sh3_graphics
         private:
         };
 
+        sh3_glprogram(const std::string& name) : programName(name){Load(name);}
```

Comment:
I'm not sure if I'm a fan of this overload. The version below still allows to ignore errors, but makes it visible to the caller that there could be one that can be checked for.